cmd/util/api.go

```diff
@@ -10,12 +10,24 @@ import (
 )
 
 func GetAPIClient(ctx context.Context) *client.APIClient {
-	return client.NewAPIClient(config.ClientAPIHost(), config.ClientAPIPort())
+	legacyTLS := client.LegacyTLSSupport(config.ClientTLSConfig())
```

Comment:
This is just type casting, right?

Reply:
Yes, they have the same structure so structural typing allows us to just convert from one to another, and if I recall correctly this was a scoping thing to avoid import cycles.